index.js

```diff
@@ -11,6 +11,7 @@ module.exports = function (movieString) {
     movie = baseName;
   }
 
-  return movie;
+  movie = movie.substring(movie.indexOf('-') + 1);
```

Comment:
don't actually know, but I did have a problem with `X-Ray` :sweat_smile:
We should probably exclude some common hyphen names.